### PR TITLE
always wait some amount of time in CA pollLoop

### PR DESCRIPTION
**Pull request by mmaxim**
Otherwise we sit in a tight loop which generates a ton of logs.

cc @jzila

### CODE REVIEW

File: go/auth/credential_authority.go [hunk 1]
```diff
@@ -226,9 +227,13 @@ func (v *CredentialAuthority) runWithCancel(body func(ctx context.Context) error
 // pollLoop() keeps running until the CA is shut down via Shutdown(). It calls Poll()
 // on the UserKeyAPIer once per iteration.
 func (v *CredentialAuthority) pollLoop() {
-	var err error
-	for err != ErrShutdown {
-		err = v.pollOnce()
+	for {
+		if err := v.pollOnce(); err != nil {
```
Comment (jzila) [inline go/auth/credential_authority.go]: So this assumes that if there is no error, `pollOnce` will have waited?

Comment (mmaxim) [inline go/auth/credential_authority.go]: Yeah, could add an additional wait I guess of a like a second if we think that is better.

Comment (jzila) [inline go/auth/credential_authority.go]: Seems like useful insurance.

**Comment by jzila** [comment]
One more bit of feedback: this means that the loop will _never_ exit if the error doesn't at some point become `ErrShutdown`. Is this desirable behavior?

**Comment by mmaxim** [comment]
I'll wait for @maxtaco to answer that one, my guess is it always eventually shows up.

**Comment by mmaxim** [comment]
K I'll just revert the last commit, the original one didn't wait.

**Comment by maxtaco** [comment]
Yeh thanks.  John if you wait if no error then you will always be slowing down rekeying. The whole point of this change was to eliminate that wait to speed up rekeying

**Comment by mmaxim** [comment]
@maxtaco Can you remind me why we didn't use the Gregor event for this?

**Comment by jzila** [comment]
@maxtaco fair enough. @mmaxim please document in a comment that it's assumed that the long poll will wait.

**Comment by maxtaco** [comment]
Guys thanks for the emergency fix.  I'm going to put up a PR which is a slightly cleaner fix.

**Pull request merged**